### PR TITLE
[RFR] Update dependencies

### PR DESCRIPTION
**Pull request by alexisjanvier**

### CODE REVIEW

**Comment by fzaninotto** [comment]
what's left for RFR?

**Comment by alexisjanvier** [comment]
@fzaninotto nothing it's in RFR (in fact I was waiting to be sure that fixing the node version would allow the tests to succeed on Travis)

File: .babelrc [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
     ]
   ],
   "plugins": [
-    "external-helpers",
-    "transform-runtime"
+    "@babel/plugin-external-helpers",
+    [
+      "@babel/plugin-transform-runtime",
+      {
+        "corejs": 2
```
Comment (zyhou) [inline .babelrc]: `corejs: 3` ? https://babeljs.io/blog/2019/03/19/7.4.0

File: package.json [hunk 2]
```diff
@@ -13,6 +13,8 @@
     "lib"
   ],
   "license": "MIT",
+  "engines" : { "node" : ">=8.10" },
+  "engineStrict" : true,
```
Comment (zyhou) [inline package.json]: "engineStrict This feature was removed in npm 3.0.0" you can remove this line

File: package.json [hunk 3]
```diff
@@ -27,35 +29,37 @@
...
-    "rollup": "~0.50.0",
-    "rollup-plugin-babel": "~3.0.2",
-    "rollup-plugin-commonjs": "~8.2.1",
-    "rollup-plugin-json": "~2.3.0",
+    "@babel/cli": "^7.0.0",
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-external-helpers": "^7.0.0",
+    "@babel/plugin-transform-runtime": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/runtime-corejs2": "~7.4.5",
```
Comment (zyhou) [inline package.json]: corejs3

File: package.json [hunk 4]
```diff
@@ -27,35 +29,37 @@
...
+    "@types/jest": "~24.0.13",
+    "babel-core": "^7.0.0-bridge.0",
```
Comment (zyhou) [inline package.json]: no needed with with jest 24 https://jestjs.io/blog/2019/01/25/jest-24-refreshing-polished-typescript-friendly

**Pull request merged**